draft-ietf-tls-tls13.md

```diff
@@ -3159,37 +3165,41 @@ The exact source of each of these secrets depends on the operational
 mode (DHE, ECDHE, PSK, etc.) and is summarized in the table below:
 
 ~~~
-    Key Exchange            Static Secret (SS)    Ephemeral Secret (ES)
-    ------------            ------------------    ---------------------
-    (EC)DHE                   Client ephemeral         Client ephemeral
-    (full handshake)       w/ server ephemeral      w/ server ephemeral
+    Key Exchange     Static Secret (SS)    Ephemeral Secret (ES)
+    ------------     ------------------    ---------------------
+    1-RTT (EC)DHE       N/A (SS = ES)        Client Ephemeral
+                                           + Server Ephemeral
```

Comment:
I'd like to leave this as-is for now, but maybe I'll clean it up in the rewrite.